content/docs/user-guide/how-to/untrack-data.md

```diff
@@ -0,0 +1,45 @@
+# How to Un-track Data
```

Comment:
I added "How to" to the titles for SEO purposes.

Reply:
Un-track feels and looks strange a bit ...

Reply:
Yes. How about "stop tracking"? Updated.